documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation/before-starting.rst

```diff
@@ -0,0 +1,6 @@
+Before starting this tutorial
+#############################
+
+You can continue this tutorial with your data. Or with the provided example data, see below.
```

Comment:
```suggestion
Please download the following example data, which contains T2w image (with fake lesions, because of the difficulty to share patient data), segmented spinal cord and segmented lesions.
```